### PR TITLE
Digital Detox: Reconnecting with Life in a Hyper-Connected World ( English, Hindi & Marathi)

### LINKED ISSUE #292: 📝[Docs]: "Digital Detox: Balancing Technology and Mental Health"
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Issue Description

"Digital Detox: Balancing Technology and Mental Health"

### Suggested Change

"Digital Detox: Balancing Technology and Mental Health"

### Rationale

_No response_

### Urgency

High

### Record

- [X] I have read the Contributing Guidelines
- [X] I'm a GSSOC'24 contributor
- [X] I want to work on this issue

### LINKED ISSUE #374: 📝[Docs]: Digital Detox: Reconnecting with Life in a Hyper-Connected World
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Issue Description

The proposed article would be for discussing the benefits of taking a break from digital devices and how it can improve mental and physical health.

### Suggested Change

English, Hindi, and Marathi

### Rationale

_No response_

### Urgency

High

### Record

- [X] I have read the Contributing Guidelines
- [X] I'm a GSSOC'24 contributor
- [X] I want to work on this issue

### PR DESCRIPTION
**Pull request by Asymtode712**
#### PR Description

This PR adds a comprehensive article titled "Digital Detox: Reconnecting with Life in a Hyper-Connected World" in English, Hindi, and Marathi. Each version of the article includes a detailed exploration of digital detox benefits, practical tips, and challenges, along with a table of contents and illustrative images to enhance reader engagement.

#### Type of Change
- [ ] Bug fix (change which fixes an issue)
- [ ] New feature (change which adds functionality)
- [X] Documentation update

#### Select your work-area
- [ ] Frontend
- [ ] Backend
- [X] Documentation
- [ ] Others

#### Related Issue
 #374 

#### Add your Work Example
📷 Add a Snapshot

#### Fixes (mention the issue number which this fixes)
Closes  #374 

#### Checklist
- [X] I have updated my branch and synced it with the project's 'develop' branch before making this PR.
- [X] I have optimized the file changes.
- [x] I have added a snapshot of my work example.
- [x] I have made a PR to the project's develop branch.

#### Undertaking
- [X] My code follows the style guidelines of this project.
- [X] I have performed a self-review of my code.
- [X] I have commented my code, particularly in hard-to-understand areas.
- [X] I have made corresponding changes to the documentation.
- [X] I have checked for plagiarism and assure its authenticity.
- [X] I have read and followed the code of conduct for this repository. I understand that violation of this undertaking may have legal consequences.

- [X] I Agree

### CODE REVIEW

**Comment by Asymtode712** [comment]
@SB2318 Please check and merge this PR

File: DataSources/English/Digital-Detox.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+- **Social Pressure:** Peer pressure to stay connected on social media can be overwhelming. Communicate your goals to friends and family and ask for their support.
+
+- **Work-Related Concerns:** Many jobs require constant digital connectivity. Set boundaries with colleagues and establish specific times for checking emails and messages.
+
+<div align="center">
+<img src="https://github.com/user-attachments/assets/bf859997-e7dc-4b9c-ab1b-0dd2e5611071">
+</div>
+
+
+## Practical Tips for a Successful Digital Detox
```
Comment (SB2318) [inline DataSources/English/Digital-Detox.md:98]: Prepare this part in form of a chart.

You can consider this as a reference:

image
 And also please include some images.

**Review by SB2318 — APPROVED** [review]
Overall Nice content! Thanks for your contribution!

**Comment by Asymtode712** [comment]
@SB2318 Done with the suggested change, please check and merge this PR

File: DataSources/English/Digital-Detox.md (hunk 1 above)
Comment (Asymtode712) [reply DataSources/English/Digital-Detox.md:98]: @SB2318 Actually, I have already added images for most of the sections, you can check them and I have also implemented the table similar to the referenced image

**Review by SB2318 — APPROVED** [review]

**Pull request merged**